tcms/testcases/tests/test_views.py

```diff
@@ -3,20 +3,20 @@
 
 import unittest
 from http import HTTPStatus
-
 from django.forms import ValidationError
 from django.test import RequestFactory
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
 
 from tcms.management.models import Priority
 from tcms.testcases.fields import MultipleEmailField
-from tcms.testcases.models import TestCase, TestCasePlan
+from tcms.testcases.models import TestCase, TestCasePlan, TestCaseStatus
 from tcms.testcases.views import get_selected_testcases
 from tcms.testruns.models import TestExecutionStatus
 from tcms.tests import (BaseCaseRun, BasePlanCase, remove_perm_from_user,
-                        user_should_have_perm)
-from tcms.tests.factories import LinkReferenceFactory, TestCaseFactory
+                        user_should_have_perm, PermissionsTestCase)
+from tcms.tests.factories import (LinkReferenceFactory, TestCaseFactory,
+                                  UserFactory, ProductFactory, TestPlanFactory, VersionFactory)
```

Comment:
There are unused imports, see flake8 results. If the line becomes too long you can also use multiple imports on separate lines.

Reply:
Done @atodorov